### PR TITLE
remove unneeded request argument

### PR DESCRIPTION
**Pull request by greg0ire**
This was overlooked in a previous refactoring.

### CODE REVIEW

**Comment by core23** [comment]
Why did you remove the `request` parameter @greg0ire ?

File: Controller/CRUDController.php [hunk 1]
```diff
@@ -283,7 +283,7 @@ public function deleteAction($id)
 
         if ($this->getRestMethod() == 'DELETE') {
             // check the csrf token
-            $this->validateCsrfToken('sonata.delete', $request);
+            $this->validateCsrfToken('sonata.delete');
```
Comment (greg0ire) [inline Controller/CRUDController.php]: @core23 : here, it is not even part of the signature anymore.

File: Controller/CRUDController.php [hunk 2]
```diff
@@ -317,7 +317,7 @@ public function deleteAction($id)
                 );
             }
 
-            return $this->redirectTo($object, $request);
+            return $this->redirectTo($object);
```
Comment (greg0ire) [inline Controller/CRUDController.php]: @core23 same here

File: Controller/CRUDController.php [hunk 3]
```diff
@@ -517,7 +517,7 @@ public function batchAction()
         $isRelevantAction = sprintf('batchAction%sIsRelevant', ucfirst($camelizedAction));
 
         if (method_exists($this, $isRelevantAction)) {
-            $nonRelevantMessage = call_user_func(array($this, $isRelevantAction), $idx, $allElements, $request);
+            $nonRelevantMessage = call_user_func(array($this, $isRelevantAction), $idx, $allElements);
```
Comment (core23) [inline Controller/CRUDController.php]: This part is used for custom batch actions @greg0ire

File: Controller/CRUDController.php [hunk 4]
```diff
@@ -578,7 +578,7 @@ public function batchAction()
             $query = null;
         }
 
-        return call_user_func(array($this, $finalAction), $query, $request);
+        return call_user_func(array($this, $finalAction), $query);
```
Comment (core23) [inline Controller/CRUDController.php]: Same here

File: Controller/CRUDController.php (hunk 3 above)
Comment (greg0ire) [inline Controller/CRUDController.php]: Woops!

**Pull request merged**